mafia/src/main/java/au/org/emii/PostgresEditor.java

```diff
@@ -158,6 +177,8 @@ public static void main(String[] args) throws Exception {
         // selection
         options.addOption("uuid", true, "etl applies to specific metadata record");
         options.addOption("all", false, "etl applies to all metadata records");
+        options.addOption("selected", true, "etl applies to selected metadata records");
+        options.addOption("isharvested", true, "etl applies to harvested or non-harvested metadata records depending upon provided value (y or n)");
```

Comment:
whats the default?

Reply:
I'd make the default non-harvested records only.

Reply:
with option isharvested
- isharvested=y : harvested records only
- isharversted=n : non-harversted records only

without option isharvested:
- all records (both harvested and non-harvested)

Reply:
Will do for the moment